Description: Correct pod NAME issues in LedgerSMB/Auth/DB.pm
 Has an incomplete module name and a comma instead of a dash in the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472535&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/Auth/DB.pm
+++ b/LedgerSMB/Auth/DB.pm
@@ -3,7 +3,7 @@
 
 =head1 NAME
 
-LedgerSMB::Auth.pm, Standard DB module.
+LedgerSMB::Auth::DB - Standard DB Authentication module.
 
 =head1 SYNOPSIS
 
